tutorials/advanced-sql-migration-tool-ora-hana/advanced-sql-migration-tool-ora-hana.md

```diff
@@ -0,0 +1,445 @@
+
+2. Oracle JDBC driver has to be installed in the host where the gateway runs -- in this case it is installed under `c:\sapdbmtk\Gateway\ora_jdbc` (`ojdbc7.jar`).
+
+    ![Select Data Source](ojdbc.png)
+
+3. Create a `.bat` file called `START_SAPDBMTK_Gateway_oracle.bat` under `c:\sapdbmtk\gateway` to start the gateway, you can use a variable to set the full path to the JDBC driver, the content of the file should be this:
+
+    ```
+    set ORACLE_JDBC=C:\sapdbmtk\gateway\ora_jdbc\ojdbc7.jar
+    start_sapdbmtk_gateway %ORACLE_JDBC%
+    ```
+
+    ![Select Data Source](gateway-bat-file.png)
+
+4. Start the gateway executing the new `START_SAPDBMTK_Gateway_oracle.bat` file -- a new command line window will be opened. Check that the `ipadress`/`hostname` and port are the right ones.
+
+    ![Select Data Source](gateway-start.png)
+
+    You are ready to run the migration tool `extract` command.
+
+5. Once the gateway is up and running execute the migration tool utility `sapdbmtk` with `extract` option, the command line parameters you have to specify are:
+
+    `sapdbmtk extract <project_folder> <ora_intance> <ora_host> <ora_port> <ora_user> <passwd>`
+
+    For example:
+
+    `c://sapdbmtk 32# sapdbmtk extract C:/tmp/Advanced_SQL_Mig_tutorial <ora_intance> XXX.XXX.XXX.XXX 1521 <oracle_user> <system_passwd>`
+
+    NOTE: the Oracle user you specify for this command needs to have read permissions on the catalogs, it will not perform any modification in the source database, SELECT_CATALOG_ROLE is enough for this purpose.
+
+    > **IMPORTANT:** windows firewall or antivirus software can prevent the migration tool connecting to the gateway, if you have connectivity issues try stopping/configuring firewall/antivirus
+
+6. When you execute `extract` option for the first time in a new project folder you will get 2 popup windows to confirm that compliance with 3rd party rights has been verified. When each of those 2 popup windows appears just read and click on **I confirm**, then rerun the command again (you will have to rerun the command twice the first time).
+
+    ![Select Data Source](compliancy-popups.png)
+
+7. Execute the `extract` command, you will see something like this.
+
+    ![Select Data Source](extract-1.png)
+
+    The `extract` command is done when you get the prompt again.
+
+    ![Select Data Source](extract-2.png)
+
+    > The complete output for the extract is located under `<project_folder>\reports` in a file named <ORA_SERVER_name\>.SQL-Extraction.Oracle-HANA.YYYYMMMDD.hhmiss.log
+
+    With `extract` command the migration tools builds its own metadata about the data model and SQL objects found in the source database. A new subfolder called `extracted` is created under the project folder, in it you will find a file for every SQL object like a stored procedure/function/trigger. and another subfolder called `catalog` storing files with information about users, tables, columns and so forth.
```

Comment:
"With the ´extract´" command..."

Either replace that period between /trigger and "and" with a comma or start with upper case "A".